llvm/lib/CodeGen/SelectionDAG/LegalizeDAG.cpp

```diff
@@ -5003,6 +5002,22 @@ void SelectionDAGLegalize::PromoteNode(SDNode *Node) {
     }
     Results.push_back(DAG.getNode(ISD::TRUNCATE, dl, OVT, Tmp1));
     break;
+  case ISD::CTLZ_ZERO_UNDEF:
```

Comment:
Remove the check for `Node->getOpcode() == ISD::CTLZ_ZERO_UNDEF` on line 4997 above?

Reply:
Done!